hw/ip/otbn/dv/otbnsim/sim/insn.py

```diff
@@ -964,6 +980,14 @@ def __init__(self, raw: int, op_vals: Dict[str, int]):
         self.wrd = op_vals['wrd']
         self.wsr = op_vals['wsr']
 
+    def pre_execute(self, state: OTBNState) -> bool:
+        if (self.wsr == 1):
```

Comment:
These parentheses aren't needed. Also, maybe it's clearer to write the `1` as `0x1` so that it looks more like a magic address?